web/realtime/realtime.go

```diff
@@ -283,7 +286,34 @@ func ws(c echo.Context) error {
 	}
 }
 
+// Notify is the API handler for POST /realtime/:doctype/:id: this route can be
+// used to send documents in the real-time without having to persist them in
+// CouchDB.
+func Notify(c echo.Context) error {
+	inst := middlewares.GetInstance(c)
+	doctype := c.Param("doctype")
+	id := c.Param("id")
+
+	if err := permission.CheckReadable(doctype); err != nil {
+		return jsonapi.BadRequest(err)
+	}
+
+	var payload couchdb.JSONDoc
+	if err := c.Bind(&payload); err != nil {
```

Comment:
What does Bind mean in this context ?

Reply:
It is a method from echo, out web framework. It fills the payload with the body of the request.